### PR TITLE
Design/calendar page

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/UI/Footer/styled.js [hunk 1]
```diff
@@ -24,13 +25,18 @@ const FooterNavItems = styled(NavLink)`
...
 
 	&:hover {
-		color: #ea2001;
-	}
-	&.active {
 		color: #ffbaa5;
 	}
+
+	&:active {
+		color: #bad;
```
Comment (JessicaLoers) [inline src/components/UI/Footer/styled.js:38]: Did you choose the hex value because of the letter sequence ` #bad` 😁

**Review by JessicaLoers — APPROVED** [review]
Well done,  @yannhn 🎉

File: src/components/UI/Footer/styled.js (hunk 1 above)
Comment (yannhn) [reply src/components/UI/Footer/styled.js:38]: haha of course! (Also maybe gregor had it in one example)

**Comment by yannhn** [comment]
Thanks for the feedback!

**Pull request merged**